flags/14.0/vttablet.txt

```diff
@@ -382,22 +271,16 @@ Usage of vttablet:
       --srv_topo_cache_refresh duration                                  how frequently to refresh the topology for cached entries (default 1s)
       --srv_topo_cache_ttl duration                                      how long to use cached entries for topology (default 1s)
       --srv_topo_timeout duration                                        topo server timeout (default 5s)
-      --stats_backend string                                             The name of the registered push-based monitoring/stats backend to use
-      --stats_combine_dimensions string                                  List of dimensions to be combined into a single "all" value in exported stats vars
-      --stats_common_tags string                                         Comma-separated list of common tags for the stats backend. It provides both label and values. Example: label1:value1,label2:value2
-      --stats_drop_variables string                                      Variables to be dropped from the list of exported variables.
-      --stats_emit_period duration                                       Interval between emitting stats to all registered backends (default 1m0s)
       --statsd_address string                                            Address for statsd client
-      --statsd_sample_rate float                                          (default 1)
-      --stderrthreshold value                                            logs at or above this threshold go to stderr (default 1)
+      --statsd_sample_rate float                                         Sample rate for statsd metrics (default 1)
+      --stderrthreshold severity                                         logs at or above this threshold go to stderr (default 1)
```

Comment:
same removed from vttablet as well, may be there were changes in statsd implementation.